interactive-clap-derive/src/derives/to_cli_args/methods/interactive_clap_attrs_cli_field.rs

```diff
@@ -97,18 +93,25 @@ impl InteractiveClapAttrsCliField {
                 }
             }
         };
+        let token_stream_args: proc_macro2::TokenStream = if !args_without_attrs.is_empty() {
+            if !named_args.is_empty() {
+                named_args
+            } else if !unnamed_args.is_empty() {
+                unnamed_args
+            } else {
+                args_without_attrs
+            }
+        } else if !named_args.is_empty() {
+            named_args
+        } else if !unnamed_args.is_empty() {
+            unnamed_args
+        } else {
+            quote!()
+        };
```

Comment:
Isn't it equivalent to?

```suggestion
        let token_stream_args: proc_macro2::TokenStream = if !named_args.is_empty() {
            named_args
        } else if !unnamed_args.is_empty() {
            unnamed_args
        } else if !args_without_attrs.is_empty() {
            args_without_attrs
        } else {
            quote!()
        };
```